content/packages-and-modules/contributing-packages-to-the-registry/about-semantic-versioning.mdx

```diff
@@ -27,7 +27,7 @@ To help developers who rely on your code, we recommend starting your package ver
 
 You can specify which update types your package can accept from dependencies in your package's `package.json` file.
 
-For example, to specify acceptable version ranges up to 1.0.4, use the following syntax:
+For example, to specify an accepted version range of 1.0.4 and up, use the following syntax:
```

Comment:
Of these 8 ranges, only 2 actually ensure “1.0.4 and up” - 1.0.3 would satisfy all the others.